aten/src/ATen/core/dispatch/OperatorEntry.h

```diff
@@ -184,7 +184,9 @@ class CAFFE2_API OperatorEntry final {
   OperatorName name_;
   c10::optional<AnnotatedSchema> schema_;
 
-  std::array<KernelFunction, static_cast<uint8_t>(DispatchKey::NumDispatchKeys)> dispatchTable_;
+  // Alias dispatch keys have entries in dispatchTable_ and they participate in the dispatch table
+  // computation, but they're not directly called in runtime.
+  std::array<KernelFunction, static_cast<uint8_t>(DispatchKey::NumDispatchKeysWithAlias)> dispatchTable_;
```

Comment:
Hmm, I find this a bit surprising. Alias keys don't participate in runtime. So shouldn't they NOT be included in the size of the dispatch table? (Since you'll never actually go to those entries.)

Reply:
Hmmm although they won't be called in runtime, I think they should still be included in the dispatch table since they participate in the computation? E.g if we register new kernels and need to update dispatch table, we'll need to look at alias keys.

Reply:
But remember we redundantly store all the keys in the flat_hash_map before. And you shouldn't be looking in `dispatchTable_` when you do computation anyway.